scripts/deploy.sh

```diff
@@ -98,6 +101,12 @@ while getopts "a:d:hH:r:s:t:u" arg; do
           NGINX_USER="nginx"
           NGINX_GROUP="nginx"
           ;;
+        opensuse|suse)
+          DISTRO="opensuse"
+          NGINX_USER="nginx"
+          NGINX_GROUP="nginx"
+          REDIS_SERVICE="redis@default"
```

Comment:
To allow parallel installation of different redis services, I would propose to use "redis@telemetrics" here.